Renderer/CKEditorRenderer.php

```diff
@@ -283,74 +310,16 @@ private function fixConfigConstants($json)
      */
     private function fixPath($path)
     {
-        $helper = $this->getAssets();
-
-        if ($helper === null) {
+        if ($this->assetsPackages === null) {
             return $path;
         }
 
-        $url = $helper->getUrl($path);
+        $url = $this->assetsPackages->getUrl($path);
 
         if (substr($path, -1) === '/' && ($position = strpos($url, '?')) !== false) {
             $url = substr($url, 0, $position);
         }
 
         return $url;
     }
-
-    /**
-     * @return JsonBuilder
-     */
-    private function getJsonBuilder()
-    {
-        return $this->container->get('ivory_ck_editor.renderer.json_builder');
-    }
-
-    /**
-     * @return string|null
-     */
-    private function getLanguage()
-    {
-        if (($request = $this->getRequest()) !== null) {
-            return $request->getLocale();
-        }
-
-        if ($this->container->hasParameter($parameter = 'locale')) {
```

Comment:
You are not providing a fallback that was given here. When you run console commands there is no request to get locale out of.